### DISCUSSION TITLE
Research Objectives

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#658] [Ideas]
@Urwumpe @bevans2000 @larsnaesbye 

So far, what we have built a rather **paper-driven** research model. 

It's not much in terms of ***exploration***, with a minor exception in that during Mineral Exploration mission settlers will collect rock samples, which in terms will enable writing some research paper on some subjects.

This [paper](https://www.academia.edu/22117201/Human_Mars_Exploration_Research_Objectives) summarizes a basic list of research topics that scientists are more interested in. It suggests visiting and exploring certain locations on Mars as well.

It would be interesting to create some **Research Missions** that target those research topics.

What do you see ?

**Comment by bevans2000** [comment]
Yes makes sense but we need to deliver #625 before adding any more Missions. The generic structure will allow us to create flexible Missions as well.

**Reply by mokun** [reply]
Sure. We are certainly raising the stakes on #625. 

One related feature in #625 I'd like to have is to incorporate some standard methods of including a list of instruments necessary for a particular mission. 

Currently, we only have containers (and EVA suit) as Equipment and they are prerequisites for a mission already.

Will you treat the handling of an instrument is a separate model we can add or is a part of this Project/Activity architecture ?

I see that the skills/experiences of utilizing instruments for data collections are even more crucial for missions.

**Comment by Urwumpe** [comment]
I agree, we need more science to take place outdoors. It appears to be too theoretical.

Also, I would maybe allow science on Mars to contribute to larger models, that is a bigger collaborative effort, like a new areological model that is build upon the sites explored.

**Reply by mokun** [reply]
> I agree, we need more science to take place outdoors. It appears to be too theoretical.
 
True, having settlers just writing research papers and earning science points is a little too abstract.

> Also, I would maybe allow science on Mars to contribute to larger models, that is a bigger collaborative effort, like a new areological model that is build upon the sites explored.

Yes we'll need to figure out what to model so that we turn the information/data that settlers collect from their missions into some sort of **actionable** initiatives or meaningful efforts or activities :)

**Reply by Paculino** [reply]
Should research have a chance of increasing yield/speed in related processes, such as aerological studies eventually increasing efficiency of mineral exploration or mining operations?

**Reply by bevans2000** [reply]
Yes completed Research should be increasing the rates of activities.

**Reply by mokun** [reply]
> Should research have a chance of increasing yield/speed in related processes, such as aerological studies eventually increasing efficiency of mineral exploration or mining operations?

Sure. we need to design a system that automatically map or matches a scientific research, or else we'll need to manually match it one by one which will be too tedious.
